### PR TITLE
Add general improvements

### PR DESCRIPTION
**Pull request by kleberbaum**
snek_blue

### CODE REVIEW

File: src/components/ContentPageSection/ContentPageSection.tsx [hunk 1]
```diff
@@ -273,9 +276,14 @@ const FullWidthImageSection = connectSection(
           //   lg: "25rem",
           //   xl: "29.375rem",
           // }}
-          
+
```
Comment (Aichnerc) [inline src/components/ContentPageSection/ContentPageSection.tsx:279]: There should be no blank line

File: src/components/Footer/FooterForms.tsx [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+      } else {
+        return {
+          name: `${day.days[0]}-${day.days[day.days.length - 1]}`,
+          open: day.open,
+          closed: day.open === "00:00" && day.close === "00:00",
+        }
+      }
+    })
+
+    console.log(`hours`, hours, days, reducedDays)
```
Comment (Aichnerc) [inline src/components/Footer/FooterForms.tsx:105]: Sure you want to keep this console.log?

File: src/components/Footer/UpperFooter.tsx [hunk 3]
```diff
@@ -1,24 +1,116 @@
...
+    [Home](/)
+    [Kontakt](/)
+    [AGB](/)
+    [Datenschutz](/)
+    [Impressum](/)
+  `
+  )
+  console.log("imprint", imprintData)
+  console.log("footerData", footerData)
+  console.log(formatOpeningHours(openingHours.data.hours))
```
Comment (Aichnerc) [inline src/components/Footer/UpperFooter.tsx:112]: Sure you want to keep this console.log?

File: src/components/Footer/useJaenNavigation.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  const regex = /\[(.*?)\]\((.*?)\)/g
+  let match
+  while ((match = regex.exec(markdown))) {
+    urls.push({
+      label: match[1],
+      to: match[2],
+    })
+  }
+
+  console.log(`urls`, urls)
```
Comment (Aichnerc) [inline src/components/Footer/useJaenNavigation.tsx:19]: Sure you want to keep this console.log?

File: src/components/Footer/FooterForms.tsx (hunk 2 above)
Comment (kleberbaum) [reply src/components/Footer/FooterForms.tsx:105]:
```suggestion
```

File: src/components/Footer/useJaenNavigation.tsx (hunk 4 above)
Comment (kleberbaum) [reply src/components/Footer/useJaenNavigation.tsx:19]:
```suggestion
```

**Review by kleberbaum — COMMENTED** [review]
Should be ok to merge after a few changes.

File: src/components/Footer/UpperFooter.tsx (hunk 3 above)
Comment (kleberbaum) [reply src/components/Footer/UpperFooter.tsx:112]:
```suggestion
```

**Review by schettn — APPROVED** [review]

**Pull request merged**